zeitgeschichte-wien.csl

```diff
@@ -0,0 +1,393 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" names-delimiter="/" demote-non-dropping-particle="sort-only" default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Zeitgeschichte Wien (German)</title>
```

Comment:
I'm seeing the journal listed as "Zeitgeschichte" everywhere. Why add the "Wien" part?